services/QuillLMS/client/app/bundles/Teacher/components/accounts/new/onboarding_tour/__tests__/tour_selection_card.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('TourSelectionCard', () => {
     expect(selectTour).toHaveBeenCalledWith(ELEMENTARY_AND_MIDDLE_SCHOOL_ELA);
   });
 
-  test('renders English Learners card and calls selectTour on click', async () => {
```

Comment:
Removing the second half for now because this tour isn't ready, so the button is disabled.